schedule/yam/agama/agama_install_with_cryptfs.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: agama_installation
```

Comment:
full disk encryption should be visible on name and description.

Reply:
done